VirtusLab-Open-Source/strapi-plugin-navigation
# fix: internalisation issues, locale versions

## Ticket

https://github.com/VirtusLab-Open-Source/strapi-plugin-navigation/issues/396

## Summary

What does this PR do/solve? 

- fixes for consistency issues in admin's panel

## Test Plan

- start the server
- add a locale (or more than one) 
- restart server
- remove at least one locale from the system
- restart server
- visit `Navigation` panel
- only currently selected locale should be displayed by UI
